cfgov/cfgov/settings/base.py

```diff
@@ -113,6 +114,9 @@
 # Absolute path to the directory static files should be collected to.
 STATIC_ROOT = os.path.join(PROJECT_ROOT, 'static')
 
+MEDIA_ROOT = os.path.join(PROJECT_ROOT, 'media')
+MEDIA_URL = '/media/'
+
```

Comment:
so what goes in the media folder? Is this were static assets imported from WP live? images etc.?